README.md

```diff
@@ -118,6 +118,15 @@ tag:
 
 This assumes that the pre-production server is otherwise ready and only needs a content sync.
 
+## Optional: Update Kielipankki Theme
+
+If you edit content in the Kielipankki theme (like popup.php or license.php) you can update it using the
+*kielipankki_theme* tag:
```

Comment:
I would maybe have called this `update_kielipankki_theme`, to indicate what the tag *does*, rather than what it *relates to*.

Reply:
yes, changed.